enterprise/utils.py

```diff
@@ -755,6 +759,10 @@ def is_course_run_available_for_enrollment(course_run):
     """
     Check if a course run is available for enrollment.
     """
+    # Check if the course run is unpublished (sometimes these sneak through)
+    if not is_course_run_published(course_run):
+        return False
+
```

Comment:
do we really need to add these lines here as well? as are already checking for `published` status in the `is_course_run_published` which we are calling from this method in the last line.

Reply:
Good catch -- I reviewed all of the dependencies on these methods and it looks like we can indeed remove this.  I've refactored the method to make it a bit more clear what is going on since I originally missed the call to `is_course_run_enrollable` in the return statement.  Thanks!